modules/vstudio/tests/vc2010/test_debug_settings.lua

```diff
@@ -101,3 +101,22 @@ foo=bar</LocalDebuggerEnvironment>
 		]]
 	end
 
+--
+-- Test Debugger Flavor
+--
+
+	function suite.debuggerFlavor_OnWindowsLocal()
+		debuggerflavor "WindowsLocalDebugger"
+		prepare()
+		test.capture [[
+<DebuggerFlavor>WindowsLocalDebugger</DebuggerFlavor>
+		]]
+	end
+
+	function suite.debuggerFlavor_OnWindowsRemote()
+		debuggerflavor "WindowsRemoteDebugger"
+		prepare()
+		test.capture [[
+<DebuggerFlavor>WindowsRemoteDebugger</DebuggerFlavor>
```

Comment:
Does this just set the "default" debugger?

Reply:
Not sure what you mean here.

Reply:
I think this is a good start for fleshing out the debugger options. If I get a chance I might try to add the remote values. :+1: